core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -104,21 +101,15 @@ public Long getFieldValue(Inode<?> o) {
     WRITE_PARENT,
   }
 
-  /** Only the root inode should have the empty string as its name. */
-  private static final String ROOT_INODE_NAME = "";
   /** Number of retries when trying to lock a path, from a given id. */
   private static final int PATH_TRAVERSAL_RETRIES = 1000;
 
-  /** The root of the entire file system. */
-  private InodeDirectory mRoot = null;
-
   /** Mount table manages the file system mount points. */
   private final MountTable mMountTable;
 
-  /** Use UniqueFieldIndex directly for ID index rather than using IndexedSet. */
-  private final FieldIndex<Inode<?>, Long> mInodes = new UniqueFieldIndex<>(ID_INDEX);
-  /** A set of inode ids representing pinned inode files. */
-  private final Set<Long> mPinnedInodeFileIds = new ConcurrentHashSet<>(64, 0.90f, 64);
+  private final TtlBucketList mTtlBuckets;
+  private final InodesView mInodes;
+  private final InodeTreePersistentState mState;
```

Comment:
can you comment on this variable, since it is an important one?

Reply:
added